internal/nodes/avi_model_nodes.go

```diff
@@ -523,11 +528,17 @@ func (v *AviVsNode) CalculateCheckSum() {
 		passthoughChecksum += passthoughChild.GetCheckSum()
 	}
 
+	// keep the order of these policies
 	policies := v.HttpPolicySetRefs
-	sort.Slice(policies, func(i, j int) bool {
-		return policies[i] < policies[j]
-	})
-	vsRefs := v.WafPolicyRef + v.AppProfileRef + utils.Stringify(policies)
+	scripts := v.VsDatascriptRefs
+
+	vsRefs := v.WafPolicyRef +
+		v.AppProfileRef +
+		utils.Stringify(policies) +
+		v.AnalyticsProfileRef +
+		v.ErrorPageProfileRef +
+		v.SSLProfileRef +
```

Comment:
Just would like to be sure that this does not affect the checksum of an existing virtualservice. If you do a stringily on an empty list, it would still add a checksum.